Source/FullScreenPokemon.ts

```diff
@@ -1319,6 +1319,34 @@ module FullScreenPokemon {
             }
         }
 
+        /**
+         * Changes the properties associated with the Player starting cycling,
+         * increasing speed, but only if the current Area allows cycling.
```

Comment:
Too descriptive. Just say you're starting the player cycling if the current area allows it (or similar).